components/CommunityEventModal.vue

```diff
@@ -101,7 +101,7 @@
             <label for="group">
               For which community?
             </label>
-            <groupSelect id="editevent" class="" @change="groupChange" />
+            <groupRememberSelect v-model="groupid" remember="editevent" />
```

Comment:
Not sure if this works correctly when editing an existing event. I created one in the playground group, but it disappeared after pressing save, maybe into moderation?

Reply:
Yes, and I later zapped it.  Create a new one if you want and I'll approve it.